skins

```diff
@@ -0,0 +1 @@
+node_modules/ckeditor/skins
```

Comment:
this symlink stuff is pretty annoying, mainly to get around the paths issues we run into.  (plugins dir is always expected to be at the same level as ckeditor.js, same with the others, but we can't just point to the npm version of ckeditor, because bender doesn't support overlapping paths.)

For example:

given `node_modules/ckeditor/*` mapped by Bender to `http://localhost:1030/apps/ckeditor/*` you can't also map `plugins/detabler/*` into `http://localhost:1030/apps/ckeditor/plugins/detabler`

so instead, we add all these links and just map the root folder in.  :/